codeforces/src/main/java/org/example/georgeAndAccommodation/Room.java

```diff
@@ -0,0 +1,19 @@
+package org.example.georgeAndAccommodation;
+
+class Room {
+    private final int accommodatedPeople;
+    private final int maxCapacity;
+
+    public Room(int howManyPeopleAccommodated, int maxCapacity) {
```

Comment:
skoro klasa pakietowa to nie ma sensu robic publicznego konstruktora, tylko pakietowy